misc/pytorch_toolkit/face_antispoofing/tests/test_face_antispoofing.py

```diff
@@ -29,16 +30,14 @@ def setUp(self):
         self.img_size = tuple(map(int, config.resize.values()))
 
     def test_export(self):
-        return
-
-        # try:
-        #     # input to inference model
-        #     dummy_input = torch.rand(size=(1, 3, *(self.img_size)), device='cpu')
-        #     self.model.eval()
-        #     torch.onnx.export(self.model, dummy_input, './mobilenetv3.onnx', verbose=False)
-        #     check_file_exist('./mobilenetv3.onnx')
-        # except ExportError:
-        #     self.fail("Exception raised while exporting to ONNX")
+        try:
```

Comment:
why do you need `try` section. If something goes wrong the test will fail anyway